### PR TITLE
Add galaxy info to meta/main.yml

### PR DESCRIPTION
**Pull request by stuartwan**
This solves #3

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -1,14 +1,23 @@
 ---
 galaxy_info:
   author: Juwai Limited
-  description:
+  description: Ansible role for installing zeromq
```
Comment (tjoelsson) [inline meta/main.yml:4]: ? libzmq

**Comment by tjoelsson** [comment]
👍

File: meta/main.yml (hunk 1 above)
Comment (stuartwan) [inline meta/main.yml:4]: I thought about it but think this is still about installing zeromq...

**Comment by agirivera** [comment]
:+1:

**Pull request merged**